tools/buildsys/src/manifest.rs

```diff
@@ -504,6 +504,7 @@ fn is_manifest_type(pkg_metadata: &PackageMetadata, manifest_type: BuildType) ->
         BuildType::Package => metadata_table.get("build-package").is_some(),
         BuildType::Kit => metadata_table.get("build-kit").is_some(),
         BuildType::Variant => metadata_table.get("build-variant").is_some(),
+        BuildType::Repack => unreachable!("Repacking is not defined in manifests"),
```

Comment:
I don't like this. Let's just return false since that is an accurate answer to the question `is_manifest_type`. A function that can panic seems out of character for us.

Edit: I have learned that this was Ben's preference so go ahead and merge it like this. I hope to make this function go away at some point with some refactoring, but we'll see...

```suggestion
        BuildType::Repack => false,
```